Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    frontend_helpers (0.5.27)
+    frontend_helpers (0.5.28)
```

Comment:
Please, check this ref for Versioning.. .

https://semver.org/spec/v2.0.0.html

Im pretty sure, we are not implementing this...

Reply:
Agree that we are not using the version as it should be, but this is a patch, so this version is well implemented 🤙